packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/class/findRootForUserCalls..st

```diff
@@ -0,0 +1,25 @@
+display profiling results
+findRootForUserCalls: aMessageTally
+
+	"Exclude the calls made by the profiler from the profiling results. 
+	Use breadth first search to find the first function call made by the user."
+	
+	| visited frontier next |
+	
+	visited := Set new. 
+	frontier := Set with: aMessageTally.
```

Comment:
I think we need to handle the case that the Method is not able to find the root Object. Maybe in this case we should through an exception ?

Reply:
done